src/main/java/org/gridsuite/study/server/networkmodificationtree/NetworkModificationNodeInfoRepositoryProxy.java

```diff
@@ -174,16 +177,21 @@ public BuildStatus getBuildStatus(AbstractNode node) {
     @Override
     public void invalidateBuildStatus(AbstractNode node, List<UUID> changedNodes) {
         NetworkModificationNode modificationNode = (NetworkModificationNode) node;
-        if (modificationNode.getBuildStatus() == BuildStatus.BUILT) {
-            updateBuildStatus(modificationNode, BuildStatus.NOT_BUILT, changedNodes);
+        if (modificationNode.getBuildStatus() != BuildStatus.BUILT) {
```

Comment:
what about the BUILDING status ? if we call invalidateBuildStatus on a node in BUILDING state, i would say it's ok to invalidate it no ?

Reply:
No modification is allowed during a node building